src/test/java/com/shopify/ShopifySdkTest.java

```diff
@@ -3182,6 +3185,64 @@ public void givenStoreWith305ProductsWhenRetrievingProductsThenReturnShopifyProd
 		}
 	}
 
+	@Test
+	public void givenSomeCustomColelctionsCreationRequestCreateAndReturnCustomCollection()	throws JsonProcessingException {
```

Comment:
Nice job on this test. Just add coverage on the other two methods you added and this looks good for us to merge in.

1. getCustomCollections(final int page, final int pageSize)
2. getCustomCollections()